apps/frontend/src/app/transformations-creator/transformations-creator.component.scss

```diff
@@ -0,0 +1 @@
+@import '~bootstrap/scss/bootstrap';
```

Comment:
You don't have to import the bootstrap style sheet in every component. This will create unnecessary, huge style files. This import is only needed in the central `styles.scss` file

Reply:
If I don't import it then the frontend looks weird. The form's size gets mismatched.
So I removed it and now I imported it in the "/apps/frontend/src/styles.scss" file